XenModel/Messages.resx

```diff
@@ -12326,6 +12326,12 @@ Please check your settings and try again.</value>
   <data name="UPDATES_WIZARD_REBOOT_NEEDED" xml:space="preserve">
     <value>{0}: This server needs to be rebooted after the update is applied.</value>
   </data>
+  <data name="UPDATES_WIZARD_REBOOT_NEEDED_LIVEPATCH_DISABLED" xml:space="preserve">
+    <value>{0}: This server needs to be rebooted after the update is applied, because live patching is disabled.</value>
+  </data>
+  <data name="UPDATES_WIZARD_REBOOT_NEEDED_LIVEPATCH_RESTRICTED" xml:space="preserve">
+    <value>{0}: This server needs to be rebooted after the update is applied, because live patching is  restricted with this server's license.</value>

```

Comment:
To match other licensing messages, how about
`This server needs to be rebooted after the update is applied, because it is not licensed for live patching.`
or better `This server is not licensed for live patching and needs to be rebooted after the update is applied.`